packages/data-controls/README.md

```diff
@@ -45,6 +45,10 @@ _Returns_
 
 -   `Object`: The control descriptor.
 
+<a name="awaitPromise" href="#awaitPromise">#</a> **awaitPromise**
+
+Undocumented declaration.
```

Comment:
While only a helper, this may be useful to document as well.

Reply:
Absolutely agreed, I'm just hesitant of polishing details until I get some deeper feedback on the approach :-)

Reply:
I personally wouldn't mind if CI explodes when it finds `Undocumented declaration` in the updated lines of README files 😅

Reply:
I followed up on this suggestion in https://github.com/WordPress/gutenberg/pull/26852